README.md

```diff
@@ -328,24 +330,23 @@ Name         | Value example | Description
 `delta_t`    |               | Elapsed time in seconds between the 'Meter Value' and the 'Previous Meter Value' measurements.
 `prv_data`   |               | Previous meter reading.
 `unit`       | "kWh"         | One of sup_units. For meter_unknown it will be a number.
-`direction`  | "export"      | Defines whether reading is based on consumption (import) or production (export). Applicable for every meter except for meter_elec.
 
 #### Important notes
 
-For backward compatibility, for service "meter_elec", interface "evt.meter.report" - reports should only support "import" direction.
+For backward compatibility, for service "meter_elec", interface "evt.meter.report" - reports should only support "import" direction. Created "evt.meter.export_report" to possibly report exported meter readings.
```

Comment:
Maybe a slightly improved version:

For backward compatibility service "meter_elec" reports imported values using "evt.meter.report" interface and exported values using "evt.meter.export_report" if applicable.

Reply:
done